src/query/storages/stream/src/stream_table.rs

```diff
@@ -392,15 +430,45 @@ impl Table for StreamTable {
         let number_of_blocks = latest_summary
             .block_count
             .abs_diff(base_summary.block_count);
-
-        Ok(Some(TableStatistics {
+        let max_stats = TableStatistics {
             num_rows: Some(num_rows),
             data_size: Some(data_size),
             data_size_compressed: Some(data_size_compressed),
             index_size: Some(index_size),
             number_of_blocks: Some(number_of_blocks),
             number_of_segments: None,
-        }))
+        };
+        // In order to join order, the statistics are predicted, which may have a large bias.
```

Comment:
how about:

// The following statistics are predicted, which may have a large bias; mainly used to determine the join order